metricbeat/module/kubernetes/state_persistentvolumeclaim/_meta/test/ksm-unit-1.8

```diff
@@ -0,0 +1,29 @@
+# HELP kube_persistentvolumeclaim_access_mode The access mode(s) specified by the persistent volume claim.
+# HELP kube_persistentvolumeclaim_info Information about persistent volume claim.
+# HELP kube_persistentvolumeclaim_labels Kubernetes labels converted to Prometheus labels.
+# HELP kube_persistentvolumeclaim_resource_requests_storage_bytes The capacity of storage requested by the persistent volume claim.
+# HELP kube_persistentvolumeclaim_status_phase The phase the persistent volume claim is currently in.
+# TYPE kube_persistentvolumeclaim_access_mode gauge
+# TYPE kube_persistentvolumeclaim_info gauge
+# TYPE kube_persistentvolumeclaim_labels gauge
+# TYPE kube_persistentvolumeclaim_resource_requests_storage_bytes gauge
+# TYPE kube_persistentvolumeclaim_status_phase gauge
+kube_persistentvolumeclaim_info{namespace="default",persistentvolumeclaim="mysql-data",storageclass="rbd",volumename="pvc-mysql-data"} 1
+kube_persistentvolumeclaim_status_phase{namespace="default",persistentvolumeclaim="mysql-data",phase="Bound"} 1
+kube_persistentvolumeclaim_status_phase{namespace="default",persistentvolumeclaim="mysql-data",phase="Lost"} 0
+kube_persistentvolumeclaim_status_phase{namespace="default",persistentvolumeclaim="mysql-data",phase="Pending"} 0
+kube_persistentvolumeclaim_resource_requests_storage_bytes{namespace="default",persistentvolumeclaim="mysql-data"} 1.073741824e+09
+kube_persistentvolumeclaim_labels{label_app="mysql-server",namespace="default",persistentvolumeclaim="mysql-data"} 1
```

Comment:
it's interesting how they included labels here when it's normally not part of the info we get. Do you know why?

Reply:
(not sure if I follow)
that labels pattern seems to be common to all main kube-state-metrics objects. Not seeing it anywhere else, nor for annotations, but it looks like the `label_keyname="value"` is here to stay.